src/main/java/org/opendevstack/provision/services/BitbucketAdapter.java

```diff
@@ -534,12 +534,12 @@ public static boolean containsGroup(JsonNode json, String group) {
         repo.setAdminGroup(project.getProjectAdminGroup());
         repo.setUserGroup(project.getProjectUserGroup());
       } else {
-        repo.setAdminGroup(globalKeyuserRoleName);
+        repo.setAdminGroup(defaultAdminGroup);
```

Comment:
This seems to not be covered with a unit test. Can you please extends it?

Reply:
Done